src/http/server.ts

```diff
@@ -706,7 +738,7 @@ export class Server {
    * @return void
    */
   protected async executeMiddlewareServerLevelAfterRequest(
-    request: any,
+    request: Drash.Http.Request,
```

Comment:
Has the doc block been updated to reflect this?

Reply:
3565ceb